src/component/tooltip/TooltipModel.ts

```diff
@@ -61,13 +61,19 @@ export interface TooltipOption extends CommonTooltipOption<TopLevelFormatterPara
     renderMode?: 'auto' | TooltipRenderMode   // TODO richText renamed canvas?
 
     /**
-     * If append popup dom to document.body
-     * Only available when renderMode is html
+     * @deprecated
+     * use appendTo: 'body' instead
      */
     appendToBody?: boolean
```

Comment:
Since the implementation of `appendToBody` is totally removed from the source code, it's better to remove this option instead of making it deprecated. This should be fixed in the future versions.

Reply:
@Ovilia  The `appendToBody` option is not removed yet. See https://github.com/apache/echarts/pull/18436/files#diff-c4bf8caece88579c9e2ae0092955d9505397b706065a6dce98f5b74c7c7209baR175.

Reply:
@plainheart OK. Thanks for correcting.